docs/marketplace/dev/feature-walkthroughs/202108.0/persistence-acl-feature-walkthrough/configuration.md

```diff
@@ -0,0 +1,276 @@
+---
+title: Configuration
+last_updated: Sep 14, 2021
+template: concept-topic-template
+---
+
+Persistence ACL feature based functionality on Propel behavior. It's possible to enable the feature in 2 ways:
+- Connect the feature to one or several tables in the database
+- Connect the feature to all tables in the database
+
+## Connect Persistence ACL feature to one or several tables
+```xml
+<?xml version="1.0"?>
+<database xmlns="spryker:schema-01" 
+          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          name="zed" 
+          xsi:schemaLocation="spryker:schema-01 https://static.spryker.com/schema-01.xsd" 
+          namespace="Orm\Zed\Merchant\Persistence"
+          package="src.Orm.Zed.Merchant.Persistence">
+    <table name="spy_merchant">
+      <behavior name="\Spryker\Zed\AclEntity\Persistence\Propel\Behavior\AclEntityBehavior"/>
+    </table>
+</database>
+```
+
+## Connect Persistence ACL feature to all tables
+```xml
+<?xml version="1.0"?>
+<database xmlns="spryker:schema-01"
+          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          name="zed"
+          xsi:schemaLocation="spryker:schema-01 https://static.spryker.com/schema-01.xsd"
+          namespace="Orm\Zed\AclEntity\Persistence"
+          package="src.Orm.Zed.AclEntity.Persistence">
+    <behavior name="\Spryker\Zed\AclEntity\Persistence\Propel\Behavior\AclEntityBehavior"/>
+</database>
+
+```
+## Feature configuration
+![Configuration entity relation diagram](https://confluence-connect.gliffy.net/embed/image/f2309504-8638-419d-abf9-783bc45c8792.png?utm_medium=live&utm_source=custom)
+The configuration, unlike the rule, is common to the system.
+The main configuration object of the module is `\Generated\Shared\Transfer\AclEntityMetadataConfigTransfer`.
+It's available for extension through the plugin system.
+All you need is create a plugin and implement `\Spryker\Zed\AclEntityExtension\Dependency\Plugin\AclEntityMetadataConfigExpanderPluginInterface`.
+
+`\Generated\Shared\Transfer\AclEntityMetadataConfigTransfer`
```

Comment:
can we have it as h3 or h4 maybe?

Reply:
### was made. looks not so good in navigation bar due the long namespaces.